inputfiles/mdn/apiDescriptions.json

```diff
@@ -48,49 +48,48 @@
+  "DOMException": "The DOMException interface represents an abnormal event (called an exception) that occurs as a result of calling a method or accessing a property of a web API.",
   "DOMImplementation": "The DOMImplementation interface represents an object providing methods which are not dependent on any particular document. Such an object is returned by the Document.implementation property.",
   "DOMParser": "The DOMParser interface provides the ability to parse XML or HTML source code from a string into a DOM Document.",
   "DOMStringList": "A type returned by some APIs which contains a list of DOMString (strings).",
-  "DOMStringMap": "Used by the dataset HTML attribute to represent data for custom attributes added to elements.",
+  "DOMStringMap": "The DOMStringMap interface is used for the HTMLElement.dataset attribute, to represent data for custom attributes added to elements.",
   "DOMTokenList": "The DOMTokenList interface represents a set of space-separated tokens. Such a set is returned by Element.classList, HTMLLinkElement.relList, HTMLAnchorElement.relList, HTMLAreaElement.relList, HTMLIframeElement.sandbox, or HTMLOutputElement.htmlFor. It is indexed beginning with 0 as with JavaScript Array objects. DOMTokenList is always case-sensitive.",
   "DataTransfer": "The DataTransfer object is used to hold the data that is being dragged during a drag and drop operation. It may hold one or more data items, each of one or more data types. For more information about drag and drop, see HTML Drag and Drop API.",
   "DataTransferItem": "The DataTransferItem object represents one drag data item. During a drag operation, each drag event has a dataTransfer property which contains a list of drag data items. Each item in the list is a DataTransferItem object.",
   "DataTransferItemList": "The DataTransferItemList object is a list of DataTransferItem objects representing items being dragged. During a drag operation, each DragEvent has a dataTransfer property and that property is a DataTransferItemList.",
   "DedicatedWorkerGlobalScope": "The DedicatedWorkerGlobalScope object (the Worker global scope) is accessible through the self keyword. Some additional global functions, namespaces objects, and constructors, not typically associated with the worker global scope, but available on it, are listed in the JavaScript Reference. See also: Functions available to workers.",
   "DelayNode": "The DelayNode interface represents a delay-line; an AudioNode audio-processing module that causes a delay between the arrival of an input data and its propagation to the output.",
-  "DeviceAcceleration": "A DeviceAcceleration object provides information about the amount of acceleration the device is experiencing along all three axes.",
+  "DeviceAcceleration": "REDIRECT  DeviceMotionEventAcceleration",
```

Comment:
can this be turned into `@deprecated` instead of, or in addition to, REDIRECT?

Reply:
I could change [`mergeApiDescriptions()`](https://github.com/microsoft/TSJS-lib-generator/blob/6c7c2ea7df1105a3bb9749f6b6ba22c7ca817c90/src/index.ts#L96) to first check if the comment from MDN starts with "REDIRECT", and if so ignore the comment and mark it as deprecated instead.

On another note, is there a reason why [`Interface`](https://github.com/microsoft/TSJS-lib-generator/blob/6c7c2ea7df1105a3bb9749f6b6ba22c7ca817c90/src/types.d.ts#L151) does not have a `deprecated` flag? I kind of expected [`markAsDeprecated()`](https://github.com/microsoft/TSJS-lib-generator/blob/6c7c2ea7df1105a3bb9749f6b6ba22c7ca817c90/src/helpers.ts#L279) to also mark the entire interface as deprecated.